### PR TITLE
Exams by Victoria Sabadan IM-22

### PR DESCRIPTION
**Pull request by witchkaa**

### CODE REVIEW

**Comment by anhrb** [comment]
Great code, done according to all requirements!

File: Tasks/password.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  const MAX = alphabet.length;
-  key = '';
-  for (let i = 0; i < length; i++) {
-    Index = Math.floor(Math.random() * MAX);
-    key = key + alphabet[Index];
+'use strict';
+
+// Generate random password.
+
+const generatePassword = (symbols, passwordLength) => {
```
Comment (tshemsedinov) [inline Tasks/password.js:5]: Naming hint: `alphabet` and `length` are ok

File: Tasks/words.js [hunk 2]
```diff
@@ -1,41 +1,19 @@
...
-      } else {
-        if (flag === true) {
-          flag = true;
-        } else {
-          flag = true;
-        }
-      }
+// Count words in a string.
+
+const words = str => {
```
Comment (tshemsedinov) [inline Tasks/words.js:5]: Use eslint before commit, we have rule in this repo and in all course codebase to add round brackets even for single-parameter functions

File: Tasks/words.js [hunk 3]
```diff
@@ -1,41 +1,19 @@
...
+  let amountOfWords = 0;
+  let charDetected = false;
+  for (const char of str) {
+    if (!charDetected && char !== ' ') {
+      charDetected = true;
+      amountOfWords++;
+    } else if (char === ' ') {
```
Comment (tshemsedinov) [inline Tasks/words.js:12]: Can be implemented without else-if but with `charDetected = !charDetected;`

**Review by tshemsedinov — APPROVED** [review]
LGTM (looks good to me)

**Pull request closed without merging**